game/data/groups.py

```diff
@@ -0,0 +1,88 @@
+from enum import Enum
+
+
+class GroupRole(Enum):
+    Unknow = "Unknown"
+    AntiAir = "AntiAir"
+    Building = "Building"
+    Naval = "Naval"
+    GroundForce = "GroundForce"
+    Defenses = "Defenses"
+    Air = "Air"
+
+
+class GroupTask(Enum):
+    # Anti Air
+    EWR = "EarlyWarningRadar"
+    AAA = "AAA"
```

Comment:
Yes, it will be introduced with the upcoming Advanced Skynet PR :) There it will be used as "Sub-Task" which allows Groups of a TGO to have a more specific sub-task. This also explains how the GroupRole should be used: For the whole logical group (yeah naming is bad.) which will be represented as TGO. To be more precise: A GroundForceGroup has a GroupTask. But one smaller group withing this groundForceGroup could have a specific task like point defence to protect the main group on its tasking. Does this makes sense?